.agents/skills/interactive-podman-desktop/scripts/launch-podman-desktop.sh

```
@@ -0,0 +1,36 @@
+    "podman-desktop.podman"
+  ]
+}
+EOF
+
+mkdir -p ~/.kube
+cp /tmp/envtest-kubeconfig ~/.kube/config
+
+DISPLAY=:99 \
+PODMAN_DESKTOP_HOME_DIR="$CUSTOM_FOLDER" \
+XDG_SESSION_TYPE=x11 \
+$(pwd)/tests/playwright/tests/PodmanDesktop/podman-desktop \
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)